config/config.go

```diff
@@ -2,16 +2,20 @@ package config
 
 import (
 	checkstypes "github.com/qonto/standards-insights/checks/aggregates"
+	"github.com/qonto/standards-insights/http"
 	rulestypes "github.com/qonto/standards-insights/rules/aggregates"
 
 	"gopkg.in/yaml.v3"
 )
 
 type Config struct {
+	HTTP      http.Config
```

Comment:
could be a great idea to add it in config.yaml (we could rename to config.example.yaml) in the way